### DISCUSSION TITLE
How to remove repository name from a custom domain hosted in Github Pages?

### DISCUSSION THREAD

**Discussion by AedenThomas** [community/community#40246] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have made a NameCheap domain and linked it to GitHub pages properly. If the site is live at a.me/resume, how can I make it just a.me and remove the repository's name from the url?

<img width="805" alt="Screenshot 2022-11-27 at 1 41 57 PM" src="https://user-images.githubusercontent.com/18345370/204210835-eec884fd-25ae-48e9-addf-466aa6327196.png">

**Comment by airtower-luna** [comment]
That sounds like the domain isn't attached to the repository itself, but instead to your user site, which then replaces your `github.io` domain for all your project sites.

If you want the site deployed from that repository at the top level, attach the domain directly to the repository.

**Reply by AedenThomas** [reply]
How do I do that?

**Reply by airtower-luna** [reply]
Both removing and adding custom domain is described here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site
